tests/test_core.py

```diff
@@ -4,4 +4,5 @@
 from __future__ import absolute_import
 
 
-from dask_ndmorph import core
+def test_import_core():
+    from dask_ndmorph import core
```

Comment:
Moved this import into a test. Really this is something we want to verify and we just had to fix a problem related to it not working ( https://github.com/dask-image/dask-ndmorph/pull/8 ). So making it a proper test so that both pytest can run without issues and we actually check that this import works.